tests/benchmarks/test_work_stealing.py

```diff
@@ -84,16 +86,21 @@ def clog(n):
 
 
 def test_work_stealing_on_straggling_worker(
-    test_name_uuid, upload_cluster_dump, benchmark_all, dask_env_variables
+    test_name_uuid,
+    upload_cluster_dump,
+    benchmark_all,
+    dask_env_variables,
+    gitlab_cluster_tags,
 ):
     with Cluster(
         name=test_name_uuid,
         n_workers=10,
         worker_vm_types=["t3.medium"],
```

Comment:
Same comment than than above, don't we want the m6 equivalent too?

Reply:
same reply, no equivalent of t3.medium